apps/web/src/pages/subscribers/SubscribersListPage.tsx

```diff
@@ -34,7 +34,7 @@ const columns: IExtendedColumn<ISubscriber>[] = [
   {
     accessor: 'createdAt',
     Header: 'Created At',
-    Cell: withCellLoading(({ row: { original } }) => format(new Date(original.createdAt), 'dd/MM/yyyy HH:mm')),
+    Cell: withCellLoading(({ row: { original } }) => format(new Date(original.createdAt), 'dd.MM.yyyy')),
```

Comment:
not sure why you also changed this formatting, we do use everywhere with "/" as a divider

Reply:
It was in the designs
will revert them

Reply:
done